hazelcast/src/main/java/com/hazelcast/jet/JobStateSnapshot.java

```diff
@@ -32,16 +33,23 @@
  */
 public final class JobStateSnapshot {
 
-    private final JetInstance instance;
+    private final HazelcastInstance instance;
     private final String name;
     private final SnapshotValidationRecord snapshotValidationRecord;
 
-    JobStateSnapshot(@Nonnull JetInstance instance, @Nonnull String name, @Nonnull SnapshotValidationRecord record) {
+    JobStateSnapshot(@Nonnull HazelcastInstance instance, @Nonnull String name, @Nonnull SnapshotValidationRecord record) {
         this.instance = instance;
         this.name = name;
         this.snapshotValidationRecord = record;
     }
 
+    @Deprecated
```

Comment:
This is private method, we don't need to keep the deprecated one.

Reply:
I removed it.